### PR TITLE
dev-styling

### PR DESCRIPTION
**Pull request by rsreenivasam-usgs**
Left align the USGS logo and headings.

### CODE REVIEW

File: src/components/HeaderUSWDSSelections.vue [hunk 1]
```diff
@@ -6,9 +6,9 @@
         <div class="usa-navbar">
           <div class="usa-logo" id="basic-logo">
             <em class="usa-logo__text"
-              ><a href="/" title="Home" aria-label="Home">{{
-                titleForSelectionHeader
-              }}</a></em
+              ><h4 href="/" title="Home" aria-label="Home">
```
Comment (ayan-usgs) [inline src/components/HeaderUSWDSSelections.vue]: I recall Mary was recommending the `h1` tag for this.

Comment (rsreenivasam-usgs) [reply src/components/HeaderUSWDSSelections.vue]: We tried out the h1 tag but we felt the text was unnecessarily large in the non mobile view.  We thought h4 was a better fit.  This is what H1 looks like.
h1

Comment (ayan-usgs) [reply src/components/HeaderUSWDSSelections.vue]: Hmm... that's kinda ugly. Is there a way to prevent it from wrapping?

Comment (rsreenivasam-usgs) [reply src/components/HeaderUSWDSSelections.vue]: I will look into it

**Pull request merged**